ingestion/src/metadata/ingestion/source/database/postgres_usage.py

```diff
@@ -26,9 +26,4 @@ class PostgresUsageSource(PostgresQueryParserSource, UsageSource):
     """
 
     sql_stmt = POSTGRES_SQL_STATEMENT
-    filters = """
-                AND (
-                     NOT s.query ILIKE 'create table %% as select %%'
-                    OR   NOT s.query ILIKE  'insert %%'
-                )
-            """
```

Comment:
What is the reason for deleting this?

Reply:
these queries are required for Usage.
https://github.com/open-metadata/OpenMetadata/issues/7387#issue-1370040388